tools/changelog-generator/changelog.js

```diff
@@ -86,6 +93,7 @@ const earliestDate = subDays(now, 7);
       }),
     )
   )
+    .filter((pr) => pr !== null) // Remove null entries
     .filter((pr) => pr.affected_files.length > 0)
     .filter((pr) => !pr.labels.includes("skip-changelog"));
```

Comment:
This line should skip PRs with `skip-changelog` applied already

Reply:
I ended up using Fabian's suggestion, it looks like it works for me. Please take a look @mheap @fabianrbz